stdlib/logging/__init__.pyi

```diff
@@ -745,7 +755,10 @@ class PercentStyle(object):  # undocumented
     if sys.version_info >= (3, 8):
         validation_pattern: Pattern[str]
     _fmt: str
-    def __init__(self, fmt: str) -> None: ...
+    if sys.version_info >= (3, 10):
+        def __init__(self, fmt: str, *, defaults: dict[str, Any] | None = ...) -> None: ...
```

Comment:
It doesn't have to be a dict, just a mapping that works with `%` formatting:

```
>>> import collections
>>> '%(a)s' % collections.ChainMap({'a':'lol'})
'lol'
```

The `defaults` argument in `Formatter.__init__` should be updated too.